tests/network/jumbo_frame/test_bond.py

```diff
@@ -141,7 +142,9 @@ def bond_bridge_attached_vma(
     name = "bond-vma"
     networks = OrderedDict()
     networks[br1bond_nad.name] = br1bond_nad.name
-    network_data_data = {"ethernets": {"eth1": {"addresses": ["10.200.1.1/24"]}}}
+    network_data_data = {
+        "ethernets": {"eth1": {"addresses": [f"{random_ipv4_address(net_seed=1, host_address=1)}/24"]}}
```

Comment:
Why not net_seed=0? I assume you just updated the existing address, which is in octet 1, but if you are already touching it, why not have it zeroed :)